db/migrate/20220128112324_add_uniqueness_index_to_library_type_name.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Library types should be unique, otherwise things get a little confusing
+class AddUniquenessIndexToLibraryTypeName < ActiveRecord::Migration[6.0]
+  def change
+    add_index :library_types, :name, unique: true
+  end
+end
```

Comment:
Database backing for the added constraint. Good practice, and Rubocop actually insists on it now.